### ISSUE TITLE
ERROR Error: Uncaught (in promise): Error: BrowserModule has already been loaded.

### ISSUE THREAD

**Issue by tahir-jamil** [pepe19000/NgxVideoListPlayer#1]
```
ERROR Error: Uncaught (in promise): Error: BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.
Error: BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.
    at new BrowserModule (platform-browser.js:1531)
    at Object.BrowserModule_Factory [as factory] (platform-browser.js:1554)
    at R3Injector.hydrate (core.js:11334)
    at R3Injector.get (core.js:11155)
    at core.js:11192
    at Set.forEach (<anonymous>)
    at R3Injector._resolveInjectorDefTypes (core.js:11192)
    at new NgModuleRef$1 (core.js:25228)
    at NgModuleFactory$1.create (core.js:25282)
    at MapSubscriber.project (router.js:3627)
    at resolvePromise (zone-evergreen.js:798)
    at resolvePromise (zone-evergreen.js:750)
    at zone-evergreen.js:860
    at ZoneDelegate.invokeTask (zone-evergreen.js:399)
    at Object.onInvokeTask (core.js:28481)
    at ZoneDelegate.invokeTask (zone-evergreen.js:398)
    at Zone.runTask (zone-evergreen.js:167)
    at drainMicroTaskQueue (zone-evergreen.js:569)
```

**Comment by pepe19000** [comment]
> ```
> ERROR Error: Uncaught (in promise): Error: BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.
> Error: BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.
>     at new BrowserModule (platform-browser.js:1531)
>     at Object.BrowserModule_Factory [as factory] (platform-browser.js:1554)
>     at R3Injector.hydrate (core.js:11334)
>     at R3Injector.get (core.js:11155)
>     at core.js:11192
>     at Set.forEach (<anonymous>)
>     at R3Injector._resolveInjectorDefTypes (core.js:11192)
>     at new NgModuleRef$1 (core.js:25228)
>     at NgModuleFactory$1.create (core.js:25282)
>     at MapSubscriber.project (router.js:3627)
>     at resolvePromise (zone-evergreen.js:798)
>     at resolvePromise (zone-evergreen.js:750)
>     at zone-evergreen.js:860
>     at ZoneDelegate.invokeTask (zone-evergreen.js:399)
>     at Object.onInvokeTask (core.js:28481)
>     at ZoneDelegate.invokeTask (zone-evergreen.js:398)
>     at Zone.runTask (zone-evergreen.js:167)
>     at drainMicroTaskQueue (zone-evergreen.js:569)
> ```

Since i do not see your Module structure, i can not help you confidently, but i can imagine your problem.
The error occurs only when import my module? Is there not error without it?

In the first round i found an unnecessary BrowserModule import in my module. I fix it, so please update package to 11.0.6 and check error still occurs or the quick fix solved your problem.

**Comment by pepe19000** [comment]
in the absence of an answer, I close the task
